ui/src/components/events/EventCard.js

```diff
@@ -11,25 +12,42 @@ import {
   Modal,
   ModalOverlay,
   ModalContent,
-  ModalHeader,
   ModalFooter,
   ModalBody,
   ModalCloseButton,
   ButtonGroup,
+  Tag,
 } from '@chakra-ui/react';
 import { RiCalendarEventFill } from 'react-icons/ri';
-import { BsClockFill } from 'react-icons/bs';
+import { BsClockFill, BsFullscreen } from 'react-icons/bs';
 import { ImLocation2 } from 'react-icons/im';
-import { useState } from 'react';
+import { FaCalendarAlt } from 'react-icons/fa';
+import { MdOutlineAddToPhotos } from 'react-icons/md';
+import { useState, useEffect } from 'react';
 import { getRenderDate } from '../helpers/eventsHelpers';
 import { DateTime } from 'luxon';
 import { generateGoogleCalendarLink } from '../helpers/eventsHelpers';
 import ReactMarkdown from 'react-markdown';
 import ChakraUIRenderer from 'chakra-ui-markdown-renderer';
+import { ColorExtractor } from 'react-color-extractor';
```

Comment:
so i tried the cors thing and it didn't work, so i think i can move the color extractor part to the announcement editor. so we just save the color when a file is uploaded instead of rendering it everytime the event page is loaded

https://stackoverflow.com/questions/29670703/how-to-use-cors-anywhere-to-reverse-proxy-and-add-cors-headers
this worked

Reply:
can we just put this on hold then